classes/02class/exercises/c02-network04/ramiray/README.md

```diff
@@ -0,0 +1,180 @@
+# c02-network04
+
+## Commands Execution Output
+
+## Subnet used in this exercise :
+
+Subnet address	
+172.16.0.0/21 - Private a							
+172.16.8.0/21 - Private b		
+172.16.16.0/20 - Private c 		
+172.16.32.0/19 - Public a	
+172.16.64.0/18 - Public b 		
+172.16.128.0/17 - Public c 
+
+
+- Commands for creating the subnets:
+```
+aws ec2 describe-vpcs --vpc-ids vpc-0351acfbc7aed9c1f
+{
+    "Vpcs": [
+        {
+            "VpcId": "vpc-0351acfbc7aed9c1f",
+            "InstanceTenancy": "default",
+            "Tags": [
+                {
+                    "Value": "devopsacademy-vpc",
+                    "Key": "Name"
+                }
+            ],
+            "CidrBlockAssociationSet": [
+                {
+                    "AssociationId": "vpc-cidr-assoc-0ab167c08ad1170cb",
+                    "CidrBlock": "172.16.0.0/16",
+                    "CidrBlockState": {
+                        "State": "associated"
+                    }
+                }
+            ],
+            "State": "available",
+            "DhcpOptionsId": "dopt-1ff7df78",
+            "CidrBlock": "172.16.0.0/16",
+            "IsDefault": false
+        }
+    ]
+}
+
+Private##:
+
+2a: 
+
+aws ec2 create-subnet --vpc-id vpc-0351acfbc7aed9c1f --availability-zone ap-southeast-2a --cidr-block 172.16.0.0/21
```

Comment:
You can add tags with
`--tag-specifications 'ResourceType=subnet,Tags=[{Key=Name,Value=private-a}]'`
if you don't want to use another command do to that later